opennms-provision/opennms-provisiond/src/main/java/org/opennms/netmgt/provision/service/ImportScheduler.java

```diff
@@ -224,19 +219,28 @@ protected void buildImportSchedule() {
             while (it.hasNext()) {
                 RequisitionDef def = it.next();
                 JobDetail detail = null;
-                CronTriggerImpl trigger = null;
+                CronTrigger trigger = null;
                 
                 try {
-                    detail = new JobDetailImpl(def.getImportName().orElse(null), JOB_GROUP, ImportJob.class, false, false);
-                    detail.getJobDataMap().put(ImportJob.URL, def.getImportUrlResource().orElse(null));
-                    detail.getJobDataMap().put(ImportJob.RESCAN_EXISTING, def.getRescanExisting());
-                    
-                    trigger = new CronTriggerImpl(def.getImportName().orElse(null), JOB_GROUP, def.getCronSchedule().orElse(null));
-                    trigger.setMisfireInstruction(CronTrigger.MISFIRE_INSTRUCTION_DO_NOTHING);
+                    detail = JobBuilder.newJob(ImportJob.class)
+                            .withIdentity(def.getImportName().orElse(null), JOB_GROUP)
+                            .requestRecovery(false)
+                            .storeDurably(false)
+                            .usingJobData(ImportJob.URL, def.getImportUrlResource().orElse(null))
+                            .usingJobData(ImportJob.RESCAN_EXISTING, def.getRescanExisting())
+                            .build();
+
+                    trigger = TriggerBuilder.newTrigger()
+                            .withIdentity(def.getImportName().orElse(null), JOB_GROUP)
+                            .forJob(detail)
+                            .usingJobData(ImportJob.URL, def.getImportUrlResource().orElse(null))
+                            .usingJobData(ImportJob.RESCAN_EXISTING, def.getRescanExisting())
+                            .withSchedule(CronScheduleBuilder
```

Comment:
Do we still need all job details or Is `forJob(detail)` sufficient here ?

Reply:
I found that just `forJob(detail)` did not add the job data map to the trigger object. The `usingJobData` lines were needed as well.

Reply:
PS: It's fixing a customer problem hence `release-29.x` :)

Reply:
Oh, i wasn't sure if this is resolving `rescanExisting` issue.  Freddy is already working on this. will add him to review.

Reply:
I did a quick test. The parameter now receive in the event at the back now. Do I still need take time to verify the behaviour matching with the document?

Reply:
@freddy61025 yeah, It's better to check expected behavior from the issue.

Reply:
dbonly behaviour of new node will be import and scan. Existing node update db and skip scan.